cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/DefaultStore.java

```diff
@@ -749,7 +766,10 @@ private boolean scanApplicationwWithReorder(ScanApplicationsRequest request,
         TransactionRunners.run(transactionRunner, context -> {
           for (int i = 0; !ids.isEmpty() && i < txBatchSize; i++) {
             ApplicationId id = ids.removeLast();
-            consumer.accept(id, getApplicationMeta(getAppMetadataStore(context), id));
```

Comment:
This is a special case dealing with noSQL db not able to sort on DESC case... Had to do the mapping here as well

Reply:
Actually my bad... This is scanning apps, where we should fetch the actual app and ignore the mapping